src/bindings/cxx/meson.build

```
@@ -63,15 +63,22 @@ foreach lib : cxx_sublibs
     endif
     subdir_file_location = join_paths(file_location, eo_file_subdir)
     foreach cxx_gen_file : cxx_pub_eo_files
+      if (package_name == 'evas' or package_name == 'efl') and cxx_gen_file in pub_eo_files_optional_ector
+        cxx_gen_file_input = cxx_gen_file
+        cxx_gen_file = '@0@'.format(cxx_gen_file)
+        cxx_gen_file = cxx_gen_file.split('/')[-1]
+      else
+        cxx_gen_file_input = join_paths(subdir_file_location, cxx_gen_file)
```

Since you're already changing this...

```suggestion
        cxx_gen_file_input = subdir_file_location / cxx_gen_file
```

---

sure.